client/src/components/Header.js

```diff
@@ -124,25 +131,25 @@ export default ({ isAuthenticated, onMenuClick }) => {
                       </li>
                     </>
                   ) : (
-                    <>
-                      <li>
-                        <NavLink activeStyle={activeStyles} to="/auth/login">
-                          Login
+                      <>
```

Comment:
isn't this too much whitespace? maybe also your eslint / prettier isn't auto formatting?

Reply:
I have prettier config set  to `true`. I actually tried to reverse them but when I save they go back to normal which is like above.